test/foundry/ExpressLaneAuction.t.sol

```diff
@@ -0,0 +1,1835 @@
+
+        // success now with the same price
+        rs.auction.resolveMultiBidAuction(bid1, rs.bid0);
+    }
+
+    function testCannotResolveReserveNotMet() public {
+        ResolveSetup memory rs = deployDepositAndBids();
+
+        bytes32 h0 = rs.auction.getBidHash(rs.biddingForRound, bidders[0].elc, minReservePrice - 1);
+        Bid memory bid0 = Bid({
+            amount: minReservePrice - 1,
+            expressLaneController: bidders[0].elc,
+            signature: sign(bidders[0].privKey, h0)
+        });
+
+        vm.expectRevert(
+            abi.encodeWithSelector(
+                ReservePriceNotMet.selector,
+                minReservePrice - 1,
+                minReservePrice
+            )
+        );
+        rs.auction.resolveMultiBidAuction(rs.bid1, bid0);
+
+        vm.expectRevert(
+            abi.encodeWithSelector(
+                ReservePriceNotMet.selector,
+                minReservePrice - 1,
+                minReservePrice
+            )
+        );
+        rs.auction.resolveSingleBidAuction(bid0);
+    }
+
+    function testCannotResolveInsufficientFunds() public {
+        ResolveSetup memory rs = deployDepositAndBids();
+
+        bytes32 h1 = rs.auction.getBidHash(
+            rs.biddingForRound,
+            bidders[1].elc,
+            bidders[1].amount * 2
+        );
+        Bid memory bid1 = Bid({
+            amount: bidders[1].amount * 2,
+            expressLaneController: bidders[1].elc,
+            signature: sign(bidders[1].privKey, h1)
+        });
+
+        vm.expectRevert(
+            abi.encodeWithSelector(
+                InsufficientBalanceAcc.selector,
+                bidders[1].addr,
+                bidders[1].amount * 2,
+                bidders[1].amount
+            )
+        );
+        rs.auction.resolveMultiBidAuction(bid1, rs.bid0);
+
+        bytes32 h0 = rs.auction.getBidHash(
+            rs.biddingForRound,
+            bidders[0].elc,
+            (bidders[0].amount * 3) / 2
+        );
+        Bid memory bid0 = Bid({
+            amount: (bidders[0].amount * 3) / 2,
+            expressLaneController: bidders[0].elc,
+            signature: sign(bidders[0].privKey, h0)
+        });
+
+        vm.expectRevert(
+            abi.encodeWithSelector(
+                InsufficientBalanceAcc.selector,
+                bidders[0].addr,
+                (bidders[0].amount * 3) / 2,
+                bidders[0].amount
+            )
+        );
+        rs.auction.resolveMultiBidAuction(rs.bid1, bid0);
+
+        vm.expectRevert(
+            abi.encodeWithSelector(
+                InsufficientBalanceAcc.selector,
+                bidders[1].addr,
+                bidders[1].amount * 2,
+                bidders[1].amount
+            )
+        );
+        rs.auction.resolveSingleBidAuction(bid1);
+    }
+
+    function testCannotResolveWrongChain() public {
+        ResolveSetup memory rs = deployDepositAndBids();
+        console.log(block.chainid);
+        return;
```

Comment:
is this expected? should we remove the test?

Reply:
fixed